src/modules/index.ts

```diff
@@ -1,4 +1,5 @@
 export * as v0_0_1 from './0.0.1'
 export * as v0_0_2 from './0.0.2'
+export * as v0_0_3 from './0.0.3'
 export * as latest from './0.0.2' // Double-exported, but simplifies connect
```

Comment:
When this will be updated?

Reply:
Should we add the TODO if we will do it later?

Reply:
Oh, actually let's do that now.

Reply:
```suggestion
export * as latest from './0.0.3' // Double-exported, but simplifies connect
```